packages/fuels/package.json

```diff
@@ -55,6 +55,7 @@
   },
   "scripts": {
     "build": "tsup --dts",
-    "build:watch": "tsup --dts --watch"
+    "build:watch": "tsup --dts --watch",
+    "prepublishOnly": "cp ../../README.md ./dist/README.md"
```

Comment:
To include files on the dist packages, just list the files on the prop inside the package.json

```json
"files": [
    "dist",
     "./README.md"
  ],
```

Reply:
Also, the readme should not be inside the dist folder after the build but in the root folder. To test it, you can run `pnpm pack`

Reply:
@luizstacio Good catch, but this will only be used at release time, and the `README` file itself is not on the `packages/fuels` directory.

@Dhaiwat10 One simple option you can try is to symlink the `/README.md` into `packages/fuels/README.md`.

> _I've suggested the use of symlinks [here](https://github.com/FuelLabs/fuels-ts/pull/694#issuecomment-1359741111) as well._

Reply:
Can we resolve this conversation since we are using a symlink?

Reply:
Fine by me. Unless @luizstacio thinks otherwise.